### PR TITLE
Add documentation link to SettingWithCopyWarning

### PR DESCRIPTION
**Pull request by gliptak**
https://github.com/pandas-dev/pandas/issues/17505

- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/indexing.rst [hunk 1]
```diff
@@ -1812,3 +1812,5 @@ This will **not** work at all, and so should be avoided
    The chained assignment warnings / exceptions are aiming to inform the user of a possibly invalid
    assignment. There may be false positives; situations where a chained assignment is inadvertently
    reported.
+
```
Comment (jreback) [inline doc/source/indexing.rst]: I would move this to the top of the section and make it a note::

File: doc/source/indexing.rst [hunk 2]
```diff
@@ -1689,6 +1689,12 @@ faster, and allows one to index *both* axes if so desired.
 Why does assignment fail when using chained indexing?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   The chained assignment warnings / exceptions are aiming to inform the user of a possibly invalid
+   assignment. There may be false positives; situations where a chained assignment is inadvertently
```
Comment (jreback) [inline doc/source/indexing.rst]: remove the last sentence; this is actually not true. All purported false positive are actually user error (at least that have been reported)

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by gliptak** [comment]
@jreback Please review.

**Comment by jreback** [comment]
@jorisvandenbossche ?

**Pull request closed without merging**